### ISSUE TITLE
Slack Hook for Administrative Actions Changes?

### ISSUE THREAD

**Issue by thetest3r** [devanooga/devanooga-meta#76]
Would adding a GitHub hook for auto publishing our administrative actions PRs into this channel be good?

**Comment by brb3** [comment]
Yea, I'm a big fan of this. Everyone is in the meta channel,. Would force us to use PRs for all changes, but I don't think that's terrible. PR and immediately accept it yourself so at least it hits the slack channel.

**Comment by JMAnderson1982** [comment]
I wouldn’t be opposed to having any PR activity for devanooga-meta published to the channel, not just admin actions.

**Comment by DigTK** [comment]
A lot of the activity already goes into the #devanooga_github channel for anyone interested. Mario's PR from earlier today showed up in there, so does this discussion :D

**Comment by brb3** [comment]
True, but that channel gets stuff for all of our repos, IIRC. I think having a hook for this repo specifically to devanooga_meta is appropriate. It has a wider affected audience. The average devanooga member doesn't need to know that I fixed a bug in the `/spongebob` command or whatever.
